### DISCUSSION TITLE
Welcome to BUREKTech Discussions!

### DISCUSSION THREAD

**Discussion by Admer456** [Admer456/ioq3-burek#22] [General]
GitHub is doing this Discussions thingy so I decided to try it out.  
Here you can generally post questions, ask for help, or discuss features, existing or imaginary.  

This is an example issue:  
- func_something is broken
- Crash when doing XYZ
- Erroneous AI behaviour when XYZ

This is an example discussion:
- What motivated you to start working on this?
- I've made a game with this thing

In case an issue is more appropriate for discussions and isn't really an issue, it will very likely be labeled with `discussion` or `question` and moved here.

**Comment by ghost** [comment]
How many Polys can this support?

**Reply by Admer456** [reply]
> How many Polys can this support?

You're not supposed to post engine questions here, but rather create a new discussion in the Q&A section. I'll answer nonetheless.

The number is tweakable, but if I recall correctly, it's 6000 out of the box. This means you can have up to 6000 world polygons rendering at once, and up to 6000 polygons per entity. You can, however, change that to anything you want. In Cirkuz 33, I changed it to 16384.
https://github.com/Admer456/ioq3-burek/blob/master/code/renderergl2/tr_local.h#L2466
